src/android/SplashScreen.java

```diff
@@ -255,6 +259,10 @@ public void run() {
                 }
                 splashDialog.setContentView(splashImageView);
                 splashDialog.setCancelable(false);
+                if (isForceFullScreen()) {
```

Comment:
Does it make sense to move this to `// check to see if the splash screen should be full screen` condition above?